src/app/helptext/sharing/smb/smb.ts

```diff
@@ -4,6 +4,7 @@ import { Validators } from "@angular/forms";
 export const helptext_sharing_smb = {
     column_name: T('Name'),
     column_path: T('Path'),
+    column_comment: T('Comment'),
```

Comment:
Network interfaces have a similar field called Description. We should standardize on one name for this type of field, and Description is clearer.